lua/telescope/builtin/diagnostics.lua

```diff
@@ -41,25 +41,42 @@ local diagnostics_to_tbl = function(opts)
     end
   end
 
+  local root_dir
```

Comment:
`opts.root_dir = opts.root_dir == true and vim.loop.cwd or opts.root_dir` I think already suffices and is more readable.

`vim.loop.cwd` is not needed here but generally favored (slightly faster and can be used in "fast"-context, IIRC).